File: app/src/main/java/com/example/multitool/EnterPinCode.java
package com.example.multitool;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toolbar;

public class EnterPinCode extends AppCompatActivity {
    private Button enter;
    private Button cancel;
    private EditText pinCode;
    private Toolbar toolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_enter_pin_code);
        enter = findViewById(R.id.button_enter);
        cancel = findViewById(R.id.button_cancel);
        pinCode = findViewById(R.id.edit_text_pin_code);
        toolbar = findViewById(R.id.toolBar);


    }


    public void hideApp(View view) {
    }
}

File: app/src/main/java/com/example/multitool/EnterPinCode.kt
package com.example.multitool

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_enter_pin_code.*

class EnterPinCode : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_enter_pin_code)
        initListener()
    }

    private fun initListener() {
    }
}